samples/generate-offline-map-with-custom-parameters/src/main/java/com/esri/arcgismaps/sample/generateofflinemapwithcustomparameters/screens/GenerateOfflineMapWithCustomParametersScreen.kt

```diff
@@ -0,0 +1,308 @@
+                    mapViewInteractionOptions = MapViewInteractionOptions(isRotateEnabled = false),
+                    mapViewProxy = mapViewModel.mapViewProxy,
+                    onLayerViewStateChanged = {
+                        mapViewModel.calculateDownloadOfflineArea()
+                    },
+                    onViewpointChangedForCenterAndScale = {
+                        mapViewModel.calculateDownloadOfflineArea()
+                    }
+                )
+                // Show bottom sheet with override parameter options
+                if (showBottomSheet) {
+                    ModalBottomSheet(
+                        modifier = Modifier.wrapContentSize(),
+                        onDismissRequest = { showBottomSheet = false },
+                        sheetState = sheetState
+                    ) {
+                        OverrideParameters(
+                            defineParameters = mapViewModel::defineGenerateOfflineMapParameters,
+                            setBottomSheetVisibility = ::setBottomSheetVisibility
+                        )
+                    }
+                }
+                // Display progress dialog while generating an offline map
+                if (mapViewModel.showJobProgressDialog) {
+                    JobLoadingDialog(title = "Generating offline map...",
+                        progress = mapViewModel.offlineMapJobProgress,
+                        cancelJobRequest = { mapViewModel.cancelOfflineMapJob() })
+                }
+
+                // Display a dialog if the sample encounters an error
+                mapViewModel.messageDialogVM.apply {
+                    if (dialogStatus) {
+                        MessageDialog(
+                            title = messageTitle, description = messageDescription, onDismissRequest = ::dismissDialog
+                        )
+                    }
+                }
+                if (mapViewModel.showResetButton) {
+                    Button(
+                        onClick = mapViewModel::reset,
+                        modifier = Modifier.align(Alignment.CenterHorizontally)
+                    ) {
+                        Text("Reset to online map")
+                    }
+                }
+            }
+        },
+        // Floating action button to show the parameter overrides bottom sheet
+        floatingActionButton = {
+            if (!showBottomSheet && !mapViewModel.showResetButton) {
+                FloatingActionButton(
+                    modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { showBottomSheet = true }
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.Settings,
+                        contentDescription = "Parameter overrides"
+                    )
+                }
+            }
+        })
+}
+
+@Composable
+fun OverrideParameters(
```

Comment:
naming suggestion 
```suggestion
fun OverrideParametersMenu(
```

or `settings`